CHANGES.md

```diff
@@ -1,5 +1,191 @@
 # Changes
 
+## stable-2.13.0
+
+This release introduces client-side policy to Linkerd, including dynamic routing
+and circuit breaking. [Gateway API](https://gateway-api.sigs.k8s.io/) HTTPRoutes
+can now be used to configure policy for outbound (client) proxies as well as
+inbound (server) proxies, by creating HTTPRoutes with Service resources as their
+`parentRef`. See the Linkerd documentation for tutorials on [dynamic request
+routing] and [circuit breaking]. New functionality for debugging HTTPRoute-based
+policy is also included in this release, including [new proxy metrics] and a new
+`linkerd diagnostics policy` CLI command.
```

Comment:
This command already existed but now it can show outbound policy (previously it could only show inbound policy)